File: Scala3/scala3-22307/trigger-program-after-minimization.scala
trait I0[T0] {
  def func0(arg0: T0): Unit

  def func1(arg0: Object): Unit = {}
}
abstract class A0 extends I1 {
  override def func0(arg0: I1): Unit = {}
}
// Java files
public interface I1 extends I0<I1> {
  public default void func0(I1 arg0) {
    throw new RuntimeException();
  }
}
public final class A1 extends A0 {
  public void func1(Object arg0) {
  }
}

File: Kotlin/KT-74174/trigger-program-after-minimization.kt
// FILE: I0.java
public interface I0<T> {
    public abstract T func(I0<T> arg0, T arg1);
}
// FILE: A0.java
public abstract class A0 implements I0<A0> {
    public A0 func(I0<Object> arg0, A0 arg1) {
        throw new RuntimeException();
    }
}

// FILE: main.kt
public abstract class A1 : A0(), I0<A0>

fun box() = "OK"